### DISCUSSION TITLE
adapter for the lathe gear, other lathe with gear bore 20 mm

### DISCUSSION THREAD

**Discussion by bestbean** [kachurovskiy/nanoels#56] [General]
Fitting lathe gear is avaiable.
There is a stl-file for a gear-to-stepper-adapter. But i have different lathe, gear bore is 20 mm, with groove 5 mm large and groove depth of 2,5 mm.
It's quite common for 9x20 lathes.  
May i ask for a stl?

**Comment by kachurovskiy** [comment]
Hi, sure, can you please point to the exact STL file you want modified? Thank you.

**Reply by bestbean** [reply]
Great! This one: https://github.com/kachurovskiy/nanoels/raw/main/h1/stepper/stepper-to-gear-adapter-12mm-10mm.stl

**Reply by kachurovskiy** [reply]
Got it.

1. How thick is your gear?
2. What's the diameter of your stepper shaft?
3. What's the depth of the flat area on the shaft?

**Reply by bestbean** [reply]
Thank you, these are basic things i forgot about.
1. 8 mm
2. 10 mm
3. 20 mm
From datasheet for stepper 23HE45-4204S: 
![stepper-axle](https://user-images.githubusercontent.com/13404673/156925086-6c480c35-a648-4007-8848-b1bf3f1d166c.png)

**Reply by kachurovskiy** [reply]
Thank you, please find the STL file here: [stepper-to-gear-adapter-20mm-10mm.zip](https://github.com/kachurovskiy/nanoels/files/8192755/stepper-to-gear-adapter-20mm-10mm.zip)

**Reply by bestbean** [reply]
Ligthning fast, thank you!
I don't understand how to fix gear on the adapter.

**Reply by kachurovskiy** [reply]
It's intended to be a friction fit. Forces on those gears are pretty small so that might be enough. If not, please post a picture of the whole thing assembled and we'll figure something out.

**Reply by bestbean** [reply]
So that might be enough also for my lathe.
Thank you a lot, you made my day!

**Comment by bestbean** [comment]
After heavy use, 2 of this 3D-printed adapters knackered.
It holds a 71 tooth modul 1 steel gear, maybe this has to to long leverage.
Before mounting T5 gears and tooth belt, i would like to try a adapter made of steel.
Ideas?

![stepper2gear20-10mm](https://user-images.githubusercontent.com/13404673/214128531-e5391a4b-0fa2-4fa9-9d06-302a638a6e97.jpg)

**Reply by kachurovskiy** [reply]
Hi @bestbean, good to hear from you again. Assuming you printed with 100% infill last time, if not - then that would go a long way making your part stronger.

Can you please post the photo of the mechanical failure for better understanding?

Some options:

- Print another copy with 100% infill from PETG
- Optionally, clean the gear thoroughly from grease and glue the new adapter in with cyanoacrylate (you can remove it later by heating to the temperature of boiling water if needed)
- Model and print the entire gear instead of just the adapter. Then the weak point would be the shaft-to-gear flat (unless if already is)
- For any of the above options, order a print on JLCPCB from a potentially stronger material
- And of course you already mentioned switching to the belt instead

Good luck!

**Reply by bestbean** [reply]
Wow, that was fast! Pretty good suggestions.
Yes,  Stepper shaft-to-adapter flat is the weak point. It is no more flat.
I've already glued the adapter to the NEMA23 axle, i let it dry until tomorrow.
I don't think it will last long.

**Reply by kachurovskiy** [reply]
Cool! If you get a chance, please do share the picture of the failure for our education :) Thanks!

**Reply by kachurovskiy** [reply]
Or you can have it printed from stainless steel for 7 euro :)

![image](https://user-images.githubusercontent.com/517919/214381035-acbd4f35-8f7b-4d2f-a70b-ffe8af1be375.png)

**Comment by bestbean** [comment]
shipping to germany, custom fees and tax  plus 42.- ...

**Reply by kachurovskiy** [reply]
Try picking `Global Standard Direct Line`? I also got an address in Germany.

![image](https://user-images.githubusercontent.com/517919/214387962-4a0f497e-60a6-458e-900c-505e013d2b7c.png)

**Reply by bestbean** [reply]
Ok, thanks!

**Comment by bestbean** [comment]
Ordered. 16,13.- all incl.

**Comment by bestbean** [comment]
Meanwhile, the adapter is here now.
I had to file it down, the bore was intended for plastic.
Material is stainless steel A4. 
Should last for long time.
![stepper-to-gear-adapter](https://user-images.githubusercontent.com/13404673/219439098-02562e0a-33e8-4945-b1db-c39f22f40318.jpg)

**Reply by kachurovskiy** [reply]
Woah, looks awesome. How accurate are the dimensions?

**Reply by bestbean** [reply]
I destroyed the stepper by pushing the adapter to hard on the spindle.
Replacement stepper already here.

**Reply by kachurovskiy** [reply]
Sorry to hear that 😔 Did you hammer it on or use the woodworking clamp?

**Reply by bestbean** [reply]
A woodworking clamp or similar would work on a stepper with dual end, what i not have.
So i heated the gear with mounted adapter and forced it onto the spindle.
Stepper now makes funny noises and moves erratically back and forward in low speed.

**Reply by bestbean** [reply]
For mounting the adapter to the new steppers spindle, i think about positioning the stepper in perpendicular position so, that i can hold the axle with a grip plier, supported by vise. So, eventually applied stronger force can't hurt the stepper, i hope.

**Comment by bestbean** [comment]
I didn't measure the dimensions. It was not much.
The *.stl was dimensioned for plastic, i presume.
The ABS-adapter from same *.stl was a perfect fit.

**Comment by bestbean** [comment]
Works finally..
I glued the the adapter with loctite 638 (esigned for the bonding of cylindrical fitting parts, particularly with narrow bond gaps approaching 0.25 mm).

The 2 steel gears are not particulary noisy, even at higher speeds. Molybdenum grease helps, as on many parts of a lathe..
